lib/components/src/blocks/ColorPalette.tsx

```diff
@@ -64,6 +79,8 @@ const SwatchColors = styled.div(({ theme }) => ({
   height: 50,
   marginBottom: 5,
   overflow: 'hidden',
+  backgroundColor: 'white',
+  backgroundImage: `repeating-linear-gradient(-45deg, #ccc, #ccc 1px, #fff 1px, #fff 16px)`,
 }));
```

Comment:
I would also add `background-clip: padding-box;` to prevent the background from showing under the transparent border.